src/blocks/temperature.rs

```diff
@@ -18,7 +18,8 @@
 //!
 //! Key | Values | Default
 //! ----|--------|--------
-//! `format` | A string to customise the output of this block. See below for available placeholders | <code>"$average avg, $max max&vert;"</code>
+//! `format` | A string to customise the output of this block when collapsed (default). See below for available placeholders | <code>" $icon "</code>
+//! `format_alt` | A string to customise the output of this block. See below for available placeholders | <code>" $icon {$average avg, $max max&vert;} "</code>
```

Comment:
I think this is a good idea, but lets do this in a separate PR

Reply:
Ok, I'll hardcode collapsed `" $icon "` format as a default and submit this change on another PR to invert the values. I've noticed in other blocks, `format_alt` is usually expanded format.

Reply:
> I'll hardcode collapsed `" $icon "` format as a default 

Why though? I think this block can function as before (for now): when block is `collapsed`, `$average`, `$max` and `$min` are just unavailable.

Reply:
I'm trying to go from:

![image](https://user-images.githubusercontent.com/749393/189485751-cf6dc290-5cbd-4ca4-b940-726ef5af4c0e.png)

```
.with_default(" $icon ")?
```

To this:

![image](https://user-images.githubusercontent.com/749393/189485813-3d960ea8-0b46-4d16-9f45-d729470464d4.png)

(looks bad with background because I was compiling some other project, usually there's no bg :smile:)

that would be `format = "$icon"` (if format was for collapsed)

We don't have `with_icon` anymore after changes here. I've now realized I still need to revert the change to available values because I had exposed them in collapsed block :see_no_evil:

Reply:
I think you can do this now with `<normal format>|$icon`

Reply:
Ahhhhhhhh, got it! You're of course right :smile: 

```toml
[[block]]
block = "temperature"

[[block]]
block = "temperature"
collapsed = false
```

```json
  [
    { "full_text": "  "},
    { "full_text": " ", "name": "1" },
    { "full_text": " 34° avg, 49° max ", "name": "1"}
  ]
```

`" $icon{ $average avg, $max max|} "` default format option renders this